src/utils/KmlUtils.js

```diff
@@ -13,6 +14,34 @@ const locationsFromPlacemark = (placemark: string): Array<Location> => {
   return locations;
 }
 
+
+
+const kmlApproximation = (kml, stop1: Stop, stop2: Stop): Array<Location> => {
+  
+  var start = 0;
+  var end = 0;
+  for (let i = 0; i < kml.locations.length; i++) {
+    if (kml.locations[i].distance(stop1.location) < kml.locations[i].distance(kml.locations[start])) {
+      start = i;
+    }
+
+    if (kml.locations[i].distance(stop2.location) < kml.locations[i].distance(kml.locations[end])) {
+      start = i;
+    }
+  }
+
+  function arrayRotate(arr, count) {

```

Comment:
Also give a one-line comment on top of or in this function to declare what it's purpose is.  `arrayRotate` is kinda vague.